tests/data/example.php

```diff
@@ -59,3 +59,8 @@ public function XY():void {
  * @todo 2023-12-14 classic multi line comment
  *   more comment data
  */
+
+// TODO: APP-123 fix it
```

Comment:
these test should be moved into a new file under `data/` to not mix up existing tests